test/watcherEx.test.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { newEnforcer } from 'casbin';
+import { RedisWatcherEx } from '../src/watcherEx';
+
+test('Test WatcherEx AddPolicy', async () => {
```

Comment:
Add test for all the default callbacks in `WatcherEx`.

Reply:
Plz see how Golang's Redis WatcherEx names it: https://github.com/casbin/redis-watcher/commit/9447bc35ede29dd2e62c23de5afb59ff8feea670

Reply:
> Add test for all the default callbacks in WatcherEx.

We don't have default callbacks?